test_source_mapping_filter.py

```
@@ -0,0 +1,110 @@
+                }
+                return { modalExists: false };
+            }
+        """)
+        
+        print(f"\n📋 Modal-Content-Analyse:")
+        print(f"   Modal existiert: {modal_content.get('modalExists', False)}")
+        
+        if modal_content.get('modalExists'):
+            print(f"   Anzahl Felder: {modal_content.get('fieldCount', 0)}")
+            print(f"   Feld-Namen: {modal_content.get('fieldNames', [])}")
+            print(f"\n✅ Filter-Ergebnisse:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)